_sass/_hackathon.scss

```diff
@@ -0,0 +1,33 @@
+
+#csh-hacks-content{
+  padding-top: 10px;
+  padding-bottom: 10px;
+  background: rgba(0,0,0,0.0);
+  & h2{
+    color: white;
+  }
+
+  & img{
+    width: 100%;
+    padding-top: 25px;
+  }
+  
+  
+
+  @media screen and (max-width: 992px){
+    padding-top: 0px;
+  }
+}
+
+.csh-hacks-wrapper{
+  padding-top: 10px;
+  padding-bottom: 10px;
```

Comment:
Why add extra padding when we already have `.spaced` on the container?

Reply:
your right it appears to be unnecessary